containers/pre_built_workers/go/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.14
+
+RUN mkdir -p /executable
+WORKDIR /executable
+
+ARG REPOSITORY=grpc/grpc-go
+ARG GITREF=master
+
+RUN git clone https://github.com/$REPOSITORY.git .
+RUN git submodule update --init
+RUN git checkout $GITREF
+
+# Save commit sha for debug use
+RUN echo 'COMMIT SHA' > GRPC_GIT_COMMIT.txt
+RUN git rev-parse $GITREF >> GRPC_GIT_COMMIT.txt
+
+# Build go excutables
+RUN go build -o /executable/bin/worker ./benchmark/worker
+
+# Copy go excuatable to a new image
+FROM debian:buster
+
+RUN mkdir -p /executable
+WORKDIR /executable
+COPY --from=0 /executable/bin/worker /executable
```

Comment:
nit: I think renaming the file from `/executable/bin/worker` in one container to `/executable/worker` in another is a tad confusing. It would be nice if we could make this consistent. They both could be `/executable/worker`. 

I assume we are using the `/executable` path to not cause conflict with `/src/workspace`? It seems a little confusing and inconsistent b/c we never provide an `/executable` mounts.

Reply:
/src/workspace does not necessarily have to be code. It was meant to be the "workspace," the current working directory for whatever we must test. Unfortunately, bazel broke this some by symlinking.

Reply:
Right, I change the path name due to the conflict. Since /src/workspace shares the volume inherited from previous stages, if we use `/src/workspace`, regardless what we have saved in the image it will became empty (inherited from clone - build- step, which in the pre-built case, we do not have anything there). 

During the process of working on pre-built image, I found out that if we copy those symlinks, the actual binaries would be copied. If really need we could do that optimization later. :) 

Thank you for helping me with the formats!